docker-compose.old-php-mysql-84.override.yml

```diff
@@ -0,0 +1,5 @@
+services:
```

Comment:
Does this need to be in the root? It feels like it would fit better inside `tools` since it's not something that needs to be looked at and thought about regularly

Reply:
Initially I had trouble getting Docker to properly use the override file, but I'm pretty sure that was from another issue I've solved since. Moved this to `tools`.